[core] Batch small changes (#21553)
mui-org/material-ui: docs/pages/api-docs/autocomplete.md

| <span class="prop-name">disabled</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | If `true`, the input will be disabled. |
| <span class="prop-name">disabledItemsFocusable</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | If `true`, will allow focus on disabled items. |
| <span class="prop-name">disableListWrap</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | If `true`, the list box in the popup will not wrap focus. |

> | <span class="prop-name">disablePortal</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | The `children` will be children in the DOM. |

Reviewer: Can we have another round on this one? cc @joshwooding

Reviewer 2: Hmm, the previous description was better. 
Maybe something like:
The `PaperComponent` will be inside the DOM hierarchy of the parent component.

for the autocomplete?

Author: If we can remove all context to the current component, it would be better, this label is shared.

Reviewer 2: Could we swap it with '`Popper` content'?

Author: Ok

Reviewer 2: So: 

The `Popper` content will be inside the DOM hierarchy of the parent component. 

?

> | <span class="prop-name">disablePortal</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | Disable the portal behavior. The children stay within its parent DOM hierarchy. |

Author: "it" refers to "children"

> | <span class="prop-name">disablePortal</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | Disable the portal behavior. The children stay within its parent DOM hierarchy. |

Reviewer 3: This doesn't make much sense because we don't know what the parent is. It makes more sense that "it" refers to the root element rendered by the AutoComplete.

> | <span class="prop-name">disablePortal</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | Disable the portal behavior. The children stay within its parent DOM hierarchy. |

Author: ```suggestion
| <span class="prop-name">disablePortal</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | Disable the portal behavior. The DOM hierarchy is left unchanged. |
```

> | <span class="prop-name">disablePortal</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | Disable the portal behavior. The children stay within its parent DOM hierarchy. |

Reviewer 3: This implies that we know what the DOM hierarchy is.

```suggestion
| <span class="prop-name">disablePortal</span> | <span class="prop-type">bool</span> | <span class="prop-default">false</span> | The `children` will be children of the combobox in the DOM. |
```

> Disable the portal behavior

Is somewhat redundant. To me this is just spelling out the prop name.
